src/olympia/activity/tests/test_views.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+import json
+
+from olympia import amo
+from olympia.activity.tests.test_serializers import LogMixin
+from olympia.amo.tests import (
+    addon_factory, user_factory, TestCase)
+from olympia.amo.urlresolvers import reverse
+from olympia.addons.models import AddonUser
+from olympia.addons.utils import generate_addon_guid
+from olympia.users.models import UserProfile
+
+
+class ReviewNotesViewSetDetailMixin(LogMixin):
+    """Tests that play with addon state and permissions. Shared between review
+    note viewset detail tests since both need to react the same way."""
+    def _test_url(self):
+        raise NotImplemented
```

Comment:
This should be `NotImplementedError`, actually. Same for the one below. I fixed the occurences in `addons` in a branch I'm working on.